### ISSUE TITLE
DateAxis() - how to set it to hour?

### ISSUE THREAD

**Issue by nikola1970** [amcharts/amcharts4#1618]
Hello, so I am having problems to set DateAxis to look at the dates per hour, currently it treats them as a whole day, but my dates are meant to be represented as hours, for example:

2019-08-19T02:00:00
2019-08-19T03:00:00
2019-08-19T04:00:00

etc... 

It represents everything as a day, and not hour by hour, but I would like to have it grouped by a day but in that day I want bars that represent hours. I can accomplish this CategoryAxis but I cannot parse dates with it...

dateAxis.baseInterval does not help, I tried it.

Basically I want this:
https://ibb.co/Kqk2swX

but getting this one:
https://ibb.co/pwPvGc3

Thanks!

**Comment by martynasma** [comment]
Not sure how that might be happening.

Try setting hourly granularity manually:

```
dateAxis.baseInterval = {
  timeUnit: "hour",
  count: 1
};
```

**Comment by nikola1970** [comment]
Hi @martynasma :)

I already tried that, when I set baseInterval to what you suggested I get this:

https://ibb.co/fNdKGLY

**Comment by martynasma** [comment]
Wanna post your chart on codepen?

**Comment by nikola1970** [comment]
@martynasma:

https://codepen.io/nikola1970/pen/ExYbKBz

**Comment by martynasma** [comment]
Thanks. Since you are using string date formats, you may need to set the date format for the chart to be able to parse it correctly:

```
chart.dateFormatter.inputDateFormat = "yyyy-MM-dd HH:mm:ss";
```

The above should do the trick.

**Comment by nikola1970** [comment]
Awesome, thanks @martynasma! Works well! :)

**Comment by martynasma** [comment]
Perfect. Feel free to close this when you're ready :)

**Comment by nikola1970** [comment]
Btw, do you know if I can npw change labels on the dateAxis? For example I would like start tick to be date, mid tick to stay 12:00 and end tick to be date as well (like "Sep 4") @martynasma

**Comment by martynasma** [comment]
Please open a new issue for different questions.

**Comment by burakgavaskargo** [comment]
Anyone can youse `chart.dateFormatter.inputDateFormat = "yyyy-MM-dd HH:mm:ss >";` if working with timezones

**Comment by souravgitt** [comment]
> Awesome, thanks @martynasma! Works well! :)

Hi,

I am using AMCharts
I have create a chart with 3 series and i want to update it hourly base. Can you help me
